django/boss/urls.py

```diff
@@ -63,6 +63,7 @@
     url(r'^v1/ingest/', include('bossingest.urls', namespace='v1')),
     url(r'^v1/collection/', include('bosscore.urls.resource_urls', namespace='v1')),
     url(r'^v1/coord/', include('bosscore.urls.coord_urls', namespace='v1')),
+    url(r'^v1/cutout/to_black/', include('bossspatialdb.urls_to_black', namespace='v1')),
```

Comment:
I would move this line up, so that it is right under the other `cutout/` url entry